crwatcher/watcher_test.go

```diff
@@ -99,6 +101,44 @@ func TestSetupHandlerAddFunc(t *testing.T) {
 	cw.handler.OnAdd(r)
 }
 
+// Test to ensure that if we are given opt in criteria we only call ResourceAdded for a resource with the specified opt
+// in criteria.
+func TestSetupHandlerAddFuncOptIn(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	mockRC := NewMockResourceController(mockCtrl)
+	cw := &CRWatcher{
+		Config: &Config{
+			OptInKey: "optIn",
```

Comment:
Since people will use tests as examples of it. Can we change `optIn` to `com.wpengine.lostromos.optIn`

Reply:
And make this change everywhere. Annotations are suppose to follow a dns naming pattern.